connectors/azure/utils/metrics.go

```diff
@@ -43,8 +50,19 @@ func CreateMetricBuilder(metric armmonitor.MetricsClientListResponse) connectors
 	var value float64 = 0
 	if len(metric.Value[0].Timeseries) == 0 || len(metric.Value[0].Timeseries[0].Data) == 0 {
 		value = 0
-	} else if metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Average != nil {
-		value = *metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Average
+	} else {
+		if metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Maximum != nil {
+			value = *metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Maximum
+		}
+		if metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Total != nil {
+			value = *metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Total
+		}
+		if metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Average != nil {
+			value = *metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Average
+		}
+		if metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Count != nil {
+			value = *metric.Value[0].Timeseries[0].Data[len(metric.Value[0].Timeseries[0].Data)-1].Count
+		}
```

Comment:
Shouldnt we be accommodating Sum also here, for whichever metrics it is applicable ?

Reply:
> Shouldnt we be accommodating Sum also here, for whichever metrics it is applicable ?

No, there is no sum in the metrics, I double-checked it during debugging. It looks like the Sum is calculated on the Azure UI side...